.githooks/lib/trufflehog-excluded.txt

```diff
@@ -1 +1,2 @@
+commands/bash/container/get-dashboard-url/get-dashboard-url
```

Comment:
This file no longer exists in the repo, but trufflehog still complains about it. Any ideas on whether we'll need to keep this line around forever?

Reply:
That is strange 🤔